app/controllers/OnlineTestController.scala

```diff
@@ -85,40 +86,40 @@ trait OnlineTestController extends BaseController {
    * @return
    */
   def completeTests(token: String) = Action.async { implicit request =>
```

Comment:
In theory these also need return types, but since they're not the focus of your PR they'll be optional bonus points only